src/FNA3D_Driver_D3D11.c

```diff
@@ -5393,10 +5671,15 @@ static void D3D11_PLATFORM_CreateSwapChain(
 	swapchainDesc.BufferUsage = DXGI_USAGE_RENDER_TARGET_OUTPUT;
 	swapchainDesc.BufferCount = 3;
 	swapchainDesc.Scaling = DXGI_MODE_SCALING_UNSPECIFIED;
-	swapchainDesc.SwapEffect = DXGI_SWAP_EFFECT_FLIP_SEQUENTIAL;
+	swapchainDesc.SwapEffect = DXGI_SWAP_EFFECT_FLIP_DISCARD;
```

Comment:
Annoyingly this was needed for UWP, I forget why... but the WinRT path can be ignored, as it's getting deleted for 24.01 anyhow.